map/map_loader/src/pointcloud_map_loader/utils.cpp

```diff
@@ -64,10 +64,14 @@ bool sphereAndBoxOverlapExists(
 bool isGridWithinQueriedArea(
   const autoware_map_msgs::msg::AreaInfo area, const PCDFileMetadata metadata)
 {
-  if (area.type == autoware_map_msgs::msg::AreaInfo::ALL_AREA) return true;
-  // Currently, the area load only supports spherical area
-  geometry_msgs::msg::Point center = area.center;
-  double radius = area.radius;
-  bool res = sphereAndBoxOverlapExists(center, radius, metadata.min, metadata.max);
+  bool res = false;
+  if (area.type == autoware_map_msgs::msg::AreaInfo::ALL_AREA) {
+    res = true;
+  } else if (area.type == autoware_map_msgs::msg::AreaInfo::CYLINDER) {
```

Comment:
My bad, this was not cylinder but a sphere. Please fix this (autoware_map_msgs as well)

Reply:
Thanks!
Correct CYLINDER to SPHERE in https://github.com/autowarefoundation/autoware.universe/pull/3162/commits/64b63d1bf9195c095b3c77e22da2762210db6e50